### PR TITLE
feat: use linea-goerli as testnet

### PR DESCRIPTION
**Pull request by philanton**

### CODE REVIEW

File: networks/testnet/assets.json [hunk 1]
```diff
@@ -1,85 +1,43 @@
...
-      "symbol": "MATIC",
-      "decimals": 18
-    },
-    {
-      "chainId": 11155111,
-      "address": "0x63FD175d3215779deBA7532fC660fA0E10c18676",
-      "name": "Duckies, the canary network for Yellow ",
+      "chainId": 59140,
+      "address": "0x4889C4d392DF005dEE5e3998D170Fa90F994699b",
+      "name": "Duckies, the canary network for Yellow",
```
Comment (mnaichuk) [inline networks/testnet/assets.json:6]: better simple Duckier or Duckies Yellow

**Review by mnaichuk — CHANGES_REQUESTED** [review]

**Review by nksazonov — APPROVED** [review]

File: networks/testnet/assets.json (hunk 1 above)
Comment (philanton) [reply networks/testnet/assets.json]: https://github.com/layer-3/clearsync/pull/171/commits/6fe457954f0763b11b2507068b90ec2373694ec5

File: networks/testnet/markets.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [
-    "spot://BTC/USDT",
-    "spot://ETH/USDT",
-    "spot://MATIC/USDT",
-    "spot://SHIB/USDT"
+    "spot://USDC/USDT",
+    "spot://LUBE/USDT",
+    "spot://WBTC/USDT",
+    "spot://LINDA/USDT"
```
Comment (mnaichuk) [inline networks/testnet/markets.json:5]: there is not trading activity for this token 
https://coinmarketcap.com/dexscan/ethereum/0x246ba530d046c0ae454b5e962a9c4ede382047bd/

Comment (mnaichuk) [inline networks/testnet/markets.json:5]: Screenshot 2024-01-18 at 13 28 02
Screenshot 2024-01-18 at 13 29 06

File: networks/testnet/markets.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 [
-    "spot://BTC/USDT",
-    "spot://ETH/USDT",
-    "spot://MATIC/USDT",
-    "spot://SHIB/USDT"
+    "spot://USDC/USDT",
+    "spot://LUBE/USDT",
```
Comment (dpatsora) [inline networks/testnet/markets.json:3]: I suggest to remote Lube and Linda from this list, until we figure out where to pull the prices

**Review by mnaichuk — APPROVED** [review]

**Pull request merged**